src/common/models/Projection.ts

```diff
@@ -491,6 +514,44 @@ export class Projection {
     );
   }
 
+  private calcCaseDensityCases(
```

Comment:
nit: Change to calcCaseDensityByCases() if you change the field name to caseDensityByCases.